SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialNetDriver.h

```diff
@@ -121,6 +124,13 @@ class SPATIALGDK_API USpatialNetDriver : public UIpNetDriver
 	UPROPERTY()
 	USnapshotManager* SnapshotManager;
 
+	// Setting ActorReplicationRateLimit in DefaultEngine.ini `[/Script/SpatialGDK.SpatialNetDriver]` section will limit the number of actors which are replicated per tick to the number specified.
```

Comment:
```suggestion
	// Limit the number of actors which are replicated per tick to the number specified.
```

This doesn't have to be `DefaultEngine.ini`, can be at different layers of the Engine ini stack and the section should be obvious to any UE4 developer.